### PR TITLE
Set up artillery

### PR DESCRIPTION
**Pull request by WDHudson**
I refactored some of the code and added an artillery testing script.

### CODE REVIEW

**Review by LachelleF — APPROVED** [review]
Cool to see an artillery implementation. :+1:

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDCProxy'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '739d9ab4aef302b2a8c010fbaa519cea8e6fNRAL',
```
Comment (LesliePajuelo) [inline newrelic.js:16]: @WDHudson invalidate your key with new relic! untrack this file and gitignore it

File: package.json [hunk 2]
```diff
@@ -22,9 +22,11 @@
   },
   "homepage": "https://github.com/PetToyCo/nick-proxy-server#readme",
```
Comment (LesliePajuelo) [inline package.json:23]: 👍

File: package.json [hunk 3]
```diff
@@ -22,9 +22,11 @@
...
   "dependencies": {
+    "artillery": "^1.6.1",
     "express": "^4.17.1",
-    "serve-static": "^1.14.1",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "newrelic": "^6.13.1",
+    "serve-static": "^1.14.1"
```
Comment (LesliePajuelo) [inline package.json:29]: what's wrong with express.static?

**Review by LesliePajuelo — CHANGES_REQUESTED** [review]
api key!

**Pull request merged**